src/osip-monitoring-controller/src/main/resources/Bundle.properties

```diff
@@ -19,3 +18,23 @@ monitoring.tank.progress.fillLevel=Fill level
 monitoring.tank.progress.temperature=Temperature
 monitoring.mixTank.motorSpeed=Motor speed
 monitoring.mixTank.color=Color
+monitoring.helpdialog.title=Help
+monitoring.helpdialog.mainTab.header=Shell
+monitoring.helpdialog.mainTab.introduction=The OSIP monitoring shell can be used to monitor the tanks, pipes and alarms of the production site simulation via the OPC UA protocol.
+monitoring.helpdialog.mainTab.layoutSection=Shell Layout
+monitoring.helpdialog.mainTab.layoutText=The layout of the monitoring shell can be divided into five parts. At first there are the information about the upper tanks of the production site. These boxes are located at the upper side of the shell and are bordered in the color of the liquid in the tank. Right of those parts is the information about the mix tank which is also bordered in the color of the mix tank. Underneath those boxes there is a textfield which is used to print alarm messages.
```

Comment:
The layout of the monitoring shell can be divided into five parts. At first **,** there are the information about the upper tanks of the production site. These boxes are located at the upper side of the shell and are bordered in the color of the liquid in the tank. **At the right side** of those parts **, there** is the information about the mix tank **. The color is shown within the box.** Underneath those boxes **,** there is a textfield which is used to print alarm messages.